in_toto/verifylib.go

```diff
@@ -201,78 +163,131 @@ func VerifyArtifacts(items []interface{},
 			expectedMaterials = item.ExpectedMaterials
 			expectedProducts = item.ExpectedProducts
 
-		default: // Something's wrong
+		default: // Something wrong
 			return fmt.Errorf("VerifyArtifacts received an item of invalid type,"+
 				" elements of passed slice 'items' must be one of 'Step' or"+
 				" 'Inspection', got: '%s'", reflect.TypeOf(item))
 		}
+
+		// Use the item's name to extract the corresponding link
 		srcLinkMb, exists := itemsMetadata[itemName]
```

Comment:
Silly question, what does `Mb` stand for in this context? I keep on thinking of MetaBlock, but I don't think this is quite right?

Reply:
Yep, Mb stands for metablock:
https://github.com/in-toto/in-toto-golang/blob/32abde0de57f3f002f55ac6319fe284cae384715/in_toto/verifylib.go#L64